examples/pre-packaged/tests/ior.yaml

```diff
@@ -0,0 +1,295 @@
+      - '    intraTestBarriers=1'
+      - '    repetitions=1'
+      - '    verbose=2'
+      - '    fsync=1'
+      - ''
+      - '### The follwing values define the nature of the benchmark test'
+      - '    api={{a_flag}}'
+      - '    filePerProc={{filecount}}'
+      - '    randomOffset={{randomize}}'
+      - '    writeFile={{write}}'
+      - '    readFile={{read}}'
+      - ''
+      - 'RUN'
+      - ''
+      - 'IOR STOP'
+
+    cmds:
+    - "echo scratch path is {{scratch.path}}"
+    - "if ! [[ -d {{scratch.path}} ]]"
+    - "then"
+    - "  echo {{scratch.path}} does not exist. Exiting."
+    - "  exit 1"
+    - "elif ! [[ -x $bloc/src/ior ]]"
+    - "then"
+    - "  echo $bloc/src/ior is not executable. Exiting."
+    - "  exit 1"
+    - "fi"
+    - "cd {{scratch.path}}"
+    - "rm -rf ior-test-{{outname}}"
+    - "mkdir ior-test-{{outname}}"
+    - "cd ior-test-{{outname}}"
+    - "{{sched.test_cmd}} {{srun_opts}} $bloc/src/ior -f $bloc/inp"
+    - "cd .."
+    - "rm -rf {{scratch.path}}/ior-test-{{outname}}"
+
+  result_parse:
+    regex:
+      max_read:
+        regex: 'Max Read: +(.*)'
+      max_write:
+        regex: 'Max Write: +(.*)'
+      fs_size:
+        regex: 'FS: (.*) +Used FS.*'
+      fs_used:
+        regex: 'Used FS: (.*)  Inodes.*'
+      inodes:
+        regex: 'Inodes: (.*) +Used Inodes.*'
+      inodes_used:
+        regex: 'Used Inodes: (.*)'
+      mem_per_node:
+        regex: 'memoryPerNode += (.*)'
+      aggregate_filesize:
+        regex: 'aggregate filesize = (.*)'
+      clients:
+        regex: 'clients += (.*)'
+      api:
+        regex: 'api += (.*)'
+      num_files:
+        regex: 'access += (.*)'
+      test_file:
+        regex: 'test filename += (.*)'
+      pattern:
+        regex: 'pattern += (.*)'
+      in_file_order:
+        regex: 'ordering in a file += (.*)'
+      inter_file_order:
+        regex: 'ordering inter file= (.*)'
+      repetitions:
+        regex: 'repetitions += (.*)'
+      transfer_size:
+        regex: 'xfersize += (.*)'
+      block_size:
+        regex: 'blocksize += (.*)'
+
+
+xroads_load1_scaling:
+  inherits_from: xroads_load1
+
+  permute_on:
+  - scratch
+  - node_count
+  - numtasks
+  - transsize
+  - mempernode
+
+  variables:
+    node_count:
+    - 1
+    - 2
+    - 3
+    - 4
+      #    - '{{ sched.nodes_avail * 0.1 }}'
+      #    - '{{ sched.nodes_avail * 0.5 }}'
+      #    - '{{ sched.nodes_avail }}'
+    numtasks:
+    - "{{ node_count }}"
+    - '{{ node_count * 2 }}'
+    - '{{ node_count * 4 }}'
+    filesize: '{{ sched.alloc_max_mem * sched.alloc_nodes * 1.5 * 1000000 }}'
+    transsize:
+    - size: "1000000"
+      spec: "1M"
+    - size: "2000000"
+      spec: "2M"
+    - size: "4000000"
+      spec: "4M"
+    - size: "8000000"
+      spec: "8M"
+    segcount: '{{ floor( filesize / ( transsize.size * numtasks ) ) }}'
+    mempernode:
+    - percent: '80%'
+      amount: '{{ floor( sched.alloc_max_mem * 0.8 ) }}'
```

Comment:
I'm pretty sure `alloc_max_mem` doesn't exist anymore. There's a `test_min_mem` now though, which is probably more appropriate.

Reply:
In fact, there shouldn't be any `alloc_` vars.

Reply:
My favorite part is that `pav show` just throws a pile of errors rather than giving me the names and descriptions of applicable variables.

Reply:
I thought it it did that right, but I guess not.